ansible_role/operator/cr.example.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: "automationbroker.io/v1alpha1"
+kind: "AutomationBroker"
+metadata:
+  name: automation-broker-operator
+  namespace: automation-broker
+spec:
+  brokerImagePullPolicy: IfNotPresent
+  createBrokerNamespace: false
+  waitForBroker: false
+################################################################################
+# Some other example overrides, in general most defaults will work.
+################################################################################
+  #brokerName: automation-broker
+  #brokerImage: "ansibleplaybookbundle/origin-ansible-service-broker"
+  #brokerClusterRole: admin
+  #brokerNodeSelector: ""
+  #brokerConfig:
```

Comment:
I would line the comment symbols up for ease of removal

Reply:
Do you mean that they should all be in the first column?

Reply:
That would work, I just usually make them line up somewhere so you can vertically select a chunk